apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -175,9 +176,18 @@ const InlineAudio = React.createClass({
         <div
           className="inline-audio"
           style={this.props.style && this.props.style.wrapper}
+          onMouseOver={function (){
+            $("#volume").css("background-color", color.cyan);
+            $(".playPause").css("background-color", color.cyan);
+          }}
+          onMouseOut={function (){
+            $("#volume").css("background-color", color.lightest_purple);
+            $(".playPause").css("background-color", color.lightest_purple);
+          }}
```

Comment:
I explored this, but because I need both buttons to have hover behavior when either one is hovered, I wasn't able to find a way to make Radium work with that.